extensions/optaplanner/runtime/src/main/java/io/quarkus/optaplanner/nativeimage/XStreamSubstitutions.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.optaplanner.nativeimage;
```

Comment:
Could you move this class to `io.quarkus.optaplanner.runtime.graal` so it's consistent with our conventions?

Reply:
Sure I can, but shouldn't `KogitoSubstitutions` (in the same folder) move too? I just followed whatever was done in that module.